src/style.css

```diff
@@ -1 +1,39 @@
-/* add styles here */
+html {
+  --star: url(/src/images/star.svg);
+  --star-activated: url(/src/images/star-active.svg);
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: var(--star);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+}
+
+.stars__star:first-child {
+  margin: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
```

Comment:
According to the conditions, yellow stars should be added using container modifier + pseudo-selector only, without extra classes. So `.stars__star` should not be a part of these selectors.